app/controllers/hub/automated_messages_controller.rb

```diff
@@ -52,14 +37,34 @@ def index
         [AutomatedMessage::Ctc2022OpenMessage, {}],
         [AutomatedMessage::PuertoRicoOpenMessage, {}],
         [AutomatedMessage::IntercomForwarding, {}],
-        [AutomatedMessage::UnmonitoredReplies, {}],
+        [AutomatedMessage::UnmonitoredReplies, { support_email: Rails.configuration.email_from[:support][:gyr] }],
         [AutomatedMessage::InProgress, {}],
       ]
 
-      @messages = messages.map do |message|
-        message_class = message[0]
-        args = message[1]
-        args.present? ? message_class.new(args) : message_class.new
+      automated_messages_and_mailers = automated_messages.map do |m|
+        message = m[0].new
+        replacement_args = {
+          body: message.email_body(**m[1]),
+          client: gyr_client,
+          preparer: User.first,
+          tax_return: gyr_client.tax_returns.first }
+        replaced_body = ReplacementParametersService.new(**replacement_args).process
```

Comment:
Personally I would prefer if the `<<Client.ClientId>>` etc things got left uninterpolated... 

otherwise we have to set up all these test objects and invoke the replacement params service and whatnot, which seems like a lot of extra code just to show fake data

Maybe instead of invoking the real ReplacementParametersService we could just take the body and escape any of the `<<` and `>>` so they become &lt;&lt; or whatever and are still visible within the iframe

Reply:
Yeah I agree the fake data does seem a little annoying to keep up with. Perhaps equally as much annoying extra code to show fake data but more comprehendible to me would be to add some fake replacement service methods like I did in this updated commit. If this is still bad though I change to escape `<<` @tgrathwell